src/router/index.js

```diff
@@ -37,5 +38,22 @@ export default route(() => {
     history: createHistory(process.env.VUE_ROUTER_BASE),
   });
 
+  Router.beforeEach(async (to, from, next) => {
+    const matches = /.+[?&]token=([^&]+)/.exec(window.location.href);
+
+    if (matches) {
+      localStorage.setItem('sessionToken', matches[1]);
+      window.location.href = window.location.href.replaceAll(/\?.+$/g, '');
+    } else if (getUserSessionToken()) {
+      initUser(getUserSessionToken())
+        .catch(() => {
+          window.location.href = process.env.LETO_MODELIZER_URL;
+        });
+      next();
```

Comment:
is it mandatory to add `next()` ? 
You added it here but not after lines 46 and 54 ?

Reply:
yes, because, 45-46 redirect to leto-modelizer not to routes of admin.